test/websockex/conn_test.exs

```diff
@@ -175,6 +175,18 @@ defmodule WebSockex.ConnTest do
       Process.sleep(50)
       assert {:error, _} = :ssl.sockname(socket)
     end
+
+    test "open_socket with custom ssl options", context do
+      ssl_options = [cacertfile: Path.join([__DIR__, "..", "support", "priv", "websockexca.cer"])]
+      conn = WebSockex.Conn.new(context.uri, ssl: ssl_options)
```

Comment:
This doesn't match what you have in your example. 😣

Reply:
Bah .. not only didn't it match the example .. the entire change was missing. no idea how i missed it, sorry for the noise. i've pushed an update.